com.unity.shadergraph/Editor/Data/Util/ListPool.cs

```diff
@@ -1,21 +1,19 @@
 using System.Collections.Generic;
+using UnityEngine.Pool;
 
 namespace UnityEditor.Graphing
 {
     static class ListPool<T>
```

Comment:
There's already a `ListPool<T>` in `UnityEngine.Pool` that we could use instead, and then get rid of this whole file.

Reply:
Done in latest commit.